app/views/layouts/application.html.erb

```diff
@@ -11,6 +11,19 @@
     <link rel="stylesheet" href="https://unicons.iconscout.com/release/v4.0.8/css/line.css">
     <%= stylesheet_link_tag "main", "data-turbo-track": "reload" %>
     <%= javascript_importmap_tags %>
+    <style>
+      <%= ".msg-#{current_user&.id}"%> {
+        background-color: #007bff;
+        border-radius: 15px;
+        padding: 10px;
+      }
+
+      <%= ".cont-#{current_user&.id}"%> {
+        display: flex;
+        justify-content: flex-end;
+        padding: 10px;
+      }
+    </style>
```

Comment:
Is there any reason to use style instead of class and CSS? Just curious 🤔

Reply:
I approved these changes just for the PR description but, always try to code with the best practices 👉🏻

Reply:
Yes, I know that the correct thing to do is to use CSS (SASS in this case), but I decided to temporarily use the style tag because it was giving me conflicts with other styles, and I didn't want to affect the changes of my other colleagues who are modifying the styles.

Still, thank you very much for your comments.